webapp-django/crashstats/symbols/management/commands/reencode-bad-sym-files.py

```diff
@@ -0,0 +1,132 @@
+import gzip
+from cStringIO import StringIO
+
+from optparse import make_option
+
+import requests
+import boto.s3.connection
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from django.template.defaultfilters import filesizeformat
+from django.utils import timezone
+
+from crashstats.symbols.models import SymbolsUpload
+
+
+class Command(BaseCommand):
+    """
+    On July 20 2015 we released an improvement to the symbols uploader so
+    that all .sym files in the zip archive bundles would be gzipped.
+    That makes them smaller.
+    However, we made a mistake in that we didn't encode the file correctly,
+    just encoding the content.
+    To correct this we have this script.
+    """
+
+    help = (
+        'Run this command to re-encode .sym files that were uploaded '
+        'without first being properly encoded with the gzip header signature.'
+    )
+
+    option_list = BaseCommand.option_list + (
+        make_option(
+            '--dry-run',
+            action='store_true',
+            dest='dry_run',
+            default=False,
+            help='Only find the files to re-encode'
+        ),
+        make_option(
+            '--first-date',
+            dest='first_date',
+            default='2015-07-20',  # the day we started compressing (132)
+            help='Upload date range start'
+        ),
+        make_option(
+            '--end-date',
+            dest='end_date',
+            default='2015-08-05',  # the day the fix was release on prod (134)
+            help='Upload date range end'
+        ),
+        make_option(
+            '--max-uploads',
+            dest='max_uploads',
+            default='0',
+            help='Max. number of SymbolsUploads to work on (default all)'
+        ),
+    )
+
+    def _parse_date(self, date):
```

Comment:
why the custom date parsing code? We have `python-dateutil` in `requirements.txt` so why not:
```
dateutil.parser.parse()
```
?

If you want to stick to stdlib for some reason why not `datetime.datetime.strptime`?